telemetry_core/src/main/java/com/newrelic/telemetry/transport/BatchDataSender.java

```diff
@@ -116,7 +116,7 @@ private Response sendPayload(byte[] payload)
           "Response from New Relic ingest API: code: {}, body: {}",
           response.getCode(),
           response.getBody());
-      if (response.getCode() == 202) {
+      if (response.getCode() == 202 || response.getCode() == 200) {
```

Comment:
might be worth a comment here about the 200 being a special case for the current event API